plugin/file/delegation_test.go

```diff
@@ -14,47 +14,23 @@ import (
 var delegationTestCases = []test.Case{
 	{
 		Qname: "a.delegated.miek.nl.", Qtype: dns.TypeTXT,
-		Ns: []dns.RR{
-			test.NS("delegated.miek.nl.	1800	IN	NS	a.delegated.miek.nl."),
-			test.NS("delegated.miek.nl.	1800	IN	NS	ns-ext.nlnetlabs.nl."),
-		},
-		Extra: []dns.RR{
-			test.A("a.delegated.miek.nl. 1800 IN A 139.162.196.78"),
-			test.AAAA("a.delegated.miek.nl. 1800 IN AAAA 2a01:7e00::f03c:91ff:fef1:6735"),
-		},
+		Ns:    delegatedMiekAuth,
```

Comment:
on second thought can you revert this? I thought all these test cases had changed but you just refactored this - I also find this less readable

Reply:
Before finding `delegation_test.go` I'd originally added the test to `lookup_test.go` where I was initially confused by the half-literal, half grouped nature of the tests. https://github.com/coredns/coredns/blob/187e1dc5122b0a6ff9f6270ed5cb355e75c3c6ba/middleware/file/lookup_test.go#L22

But once I understood it, I did find the pre-grouped variable carried intent better - and brought it into `delegation_test.go`.

But I understand that it may be found less readable.